dtfs-central-api/src/services/portal/facility-amendment.service.ts

```diff
@@ -14,32 +14,30 @@ import {
   PORTAL_AMENDMENT_INPROGRESS_STATUSES,
 } from '@ukef/dtfs2-common';
 import { ObjectId } from 'mongodb';
-import { cloneDeep, remove } from 'lodash';
+import { cloneDeep } from 'lodash';
 import { findOneUser } from '../../v1/controllers/user/get-user.controller';
 import { TfmFacilitiesRepo } from '../../repositories/tfm-facilities-repo';
 import { EligibilityCriteriaAmendmentsRepo } from '../../repositories/portal/eligibility-criteria-amendments.repo';
 import { findOneFacility } from '../../v1/controllers/portal/facility/get-facility.controller';
 
 export class PortalFacilityAmendmentService {
   /**
-   * Checks if there are any other portal amendments under way on a deal, throws an error if there are.
+   * Checks if there are any other portal amendment in progress on a deal, throws an error if there are.
```

Comment:
```suggestion
   * Checks if there are any other portal amendments in progress on a deal, throws an error if there are.
```

Reply:
```suggestion
   * Checks if there is any portal amendment in progress on a deal, throws an error if there is.
```
I changed to this. Only one amendment in progress will be on a deal